scripts/us_census/acs5yr/subject_tables/jsonSpec.md

```diff
@@ -0,0 +1,203 @@
+			"statType": "medianValue",
+			"unit": "USDollar"
+		},
+
+		"_DEFAULT": {
+			"measuredProperty": "count",
+			"statType": "measuredValue"
+		}
+   }
+
+```
+
+#### `pvs`
+The `pvs` section describes the additional constraint properties that are added to define a stat-var based on column tokens.
+
+```json
+  "pvs":{
+		"healthInsurance": {
+			"Total Uninsured": "NoHealthInsurance",
+			"Uninsured Population": "NoHealthInsurance",
+			"Total Uninsured MOE": "NoHealthInsurance"
+		}
+}
+
+```
+
+#### `universePVs` [optional]
+Defines a set of additional properties and values to the stat-var based on a specific `populationType` and list of constraint properties.
+
+In the example, if the `populationType` is a `Person` and the constraint property is `healthInsurance`, the additional property-values for `armedForcesStatus` and `institutionalization` are added. If there are multiple constrainProperties, the additionals pvs (dependent PVs) are added with a logical `AND` condition.
+
+```json
+  "universePvs":[
+		{
+			"populationType": "Person",
+			"constraintProperties": ["healthInsurance"],
+			"dependentPVs": {
+				"armedForcesStatus": "Civilian",
+				"institutionalization": "USC_NonInstitutionalized"
+			}
+		}
+   ]
+```
+
+
+#### `inferedSpec` [optional]
+Adds additional property-value pairs to a stat-var node based on the inference of an existing property in that stat-var node.
+
+For instance, in the example, if the stat-var node has a property `employment` defined, additional property-values for `employmentStatus` and `age` are added to the stat-var node.
+
+```json
+  "inferredSpec":{
+		"employmentStatus": {
+			"age": "[16 - Years]"
+		},
+		"employment": {
+			"employmentStatus": "BLS_InLaborForce",
+			"age": "[16 - Years]"
+		}
+   }
+```
+
+#### `enumSpecializations` [optional]
+This section is not used in the stat-var node but is used to process the column name for which a stat-var node is to be generated.
+
+
+```json
+  "enumSpecializations":{
+		"Under 6 years": "Under 18 years, Under 19 years",
+		"6 to 17 years": "Under 18 years",
+		"6 to 18 years": "Under 19 years",
+		"18 to 24 years": "18 to 64 years",
+		"24 to 34 years": "18 to 64 years",
+		"19 to 25 years": "19 to 64 years",
+		"26 to 34 years": "19 to 64 years",
+		"35 to 44 years": "18 to 64 years, 19 to 64 years",
+		"45 to 54 years": "18 to 64 years, 19 to 64 years",
+		"55 to 64 years": "18 to 64 years, 19 to 64 years",
+		"65 to 75 years": "65 years and older"
+   }
+```
+For instance, if the column name is *Estimate!!Total Uninsured!!Total civilian noninstitutionalized population!!AGE!!Under 19 years!!Under 6 years*, the column token *Under 6 years* is a specialization of *Under 19 years*. So, while generating the stat-var node for this column we would like to keep only the specialization, which will mean the stat-var node generated will be for the column *Estimate!!Total Uninsured!!Total civilian noninstitutionalized population!!AGE!!Under 6 years*
```

Comment:
The code in #488 seems to drop the whole column:
*Estimate!!Total Uninsured!!Total civilian noninstitutionalized population!!AGE!!Under 19 years"
rather than just a token in the column.